### ISSUE TITLE
"Access is denied" on localStorage in IE10/11 if user has no permissions on LocalLow folder

### ISSUE THREAD

**Issue by Reinmar** [WebSpellChecker/ckeditor-plugin-scayt#100]
In Windows, if user has no permissions on c:\users<username>\AppData\LocalLow folder, cke editor will not be loaded with error:

```
Access is denied
```

in ckeditor.js file on any request to localStorage, like:

```
localStorage&&localStorage.removeItem("scayt_0_lang")
```

Based on http://dev.ckeditor.com/ticket/13471

**Comment by WebSpellChecker** [comment]
Thank you for reporting the problem. Our QA team will verify it tomorrow (July 1, 2015)

**Comment by KremYura** [comment]
It looks like to be a problem with the settings on the C:\Users\<username>\Appdata\LocalLow folder in user profile.

To solve the problem, try run the following command in a command prompt:
```
icacls %userprofile%\Appdata\LocalLow /t /setintegritylevel (OI)(CI)L
```

**Comment by Reinmar** [comment]
One thing is what the system administrator can do. The second thing is what SCAYT should do – it should not fail to work when local storage is not available. We've just seen similar issues on iOS when Safari was in private mode (we'll report a new ticket for this).

**Comment by Reinmar** [comment]
The other ticket reported in #103.

**Comment by WebSpellChecker** [comment]
We will look for the fix during next development sprint that is started on Monday (August 17). We'll update you with the status.

**Comment by wsc-bug-hunter** [comment]
Hello everyone,
I hope you are doing great!

I have tried to reproduce the issue and it seems that it has been fixed.  Could someone please confirm this?

Regards,
WebSpellChecker

**Comment by wsc-julia-shaptala** [comment]
Hi all,

Please be informed that this issue has been fixed in v4.9.4.

Sincerely,
